pkg/ssh/ssh.go

```diff
@@ -130,6 +133,8 @@ func NewClient(cfg *Config, logger log.Logger, km *KeyManager) *Client {
 func (s *Client) starting(ctx context.Context) error {
 	level.Info(s.logger).Log("msg", "starting ssh client")
 
+	prometheus.MustRegister(s.metrics.sshRestartsCount)
```

Comment:
I'd like to be consistent and move all of the metrics registrations out to `main` like we've done with pdcClient